ui_components.py

```
@@ -0,0 +1,906 @@
+    
+    def _start_live_updates(self):
+        """Startet Live-Updates der Statistiken."""
+        def update_stats():
+            while True:
+                try:
+                    import psutil
+                    
+                    # CPU-Nutzung
+                    cpu_percent = psutil.cpu_percent(interval=1)
+                    self.after(0, lambda: self._update_stat_display(
+                        self.cpu_frame.value_label, f"{cpu_percent:.1f}%"
```

Ruff (0.12.2)

Function definition does not bind loop variable `cpu_percent`

(B023)